TOOLS/gen-interface-changes.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+# Generate a new interface-changes.rst based on the entries in
+# the interface-changes directory.
+
+#
+# This file is part of mpv.
+#
+# mpv is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# mpv is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with mpv.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+import pathlib
+import sys
+
+def add_new_entries(docs_dir, out):
+    changes_dir = pathlib.Path(docs_dir) / "interface-changes"
+    files = [f for f in pathlib.Path(changes_dir).glob("*.txt") if f.is_file()]
+    files.sort()
```

Comment:
so they're not sorted chronologically?
IMO that's desirable

Reply:
Could sort by file creation time. I think that works assuming the file creation time is the PR gets merged into master and not when it was created in the branch.

Reply:
I don't think git sets file creation times.
You probably have to call out to `git log` or whatever to determine the actual date.

Reply:
Should work now hopefully.